Monika After Story/game/script-topics.rpy

```diff
@@ -1124,6 +1124,53 @@ label monika_wine:
     m 2hksdlb "...Gosh, what am I talking about this for?"
     m "I don't condone underage drinking!"
     m 2eua "I mean, I've never drank or anything, so...yeah."
+    m 3eua "Speaking of which, I hear different parts of the world have different minimum drinking ages."
+    m "Would we both be able to have a drink where you're from?"
+    menu:
+        "Yes.":
+            m 1hua "That's great!"
+            m 1hub "Another thing to look forward to when we can finally be together!"
+            m 1hubfa "Sitting together, having a drink as we both ponder about life..."
+            m 1dkbfa "Oh gosh... I'm imagining being able to share a lovely drink whenever the mood gets romantic."
+            m 1hksdlb "Of course, we probably shouldn't get {i}too{/i} carried away..."
+            m 1eka "The finest wine isn't exactly cheap if we get too attached to sharing a drink."
+            show monika 5eua at t11 zorder 2 with dissolve
```

Comment:
Please use `MAS_MONIKA_Z` for zorders, otherwise she just goes behind the classroom.